File: shamelagpt-android/app/src/main/java/com/shamelagpt/android/data/remote/dto/ConversationResponse.kt
package com.shamelagpt.android.data.remote.dto

import com.google.gson.annotations.SerializedName

/**
 * Minimal conversation response model to mirror backend shape.
 */
data class ConversationResponse(
    val id: String,
    @SerializedName("thread_id")
    val threadId: String?,
    val title: String?,
    @SerializedName("created_at")
    val createdAt: String?,
    @SerializedName("updated_at")
    val updatedAt: String?
)

File: shamelagpt-ios/shamelagpt/Core/Networking/Models/ConversationResponse.swift
//
//  ConversationResponse.swift
//  ShamelaGPT
//
//  Created by Codex on 05/12/2025.
//

import Foundation

struct ConversationResponse: Codable, Equatable {
    let id: String
    let threadId: String?
    let title: String?
    let createdAt: String?
    let updatedAt: String?
}
